rewrite-javascript/build.gradle.kts

```diff
@@ -56,6 +56,17 @@ extensions.configure<NodeExtension> {
 }
 
 val npmTest = tasks.named("npm_test")
+npmTest.configure {
+    inputs.files(fileTree("rewrite") {
+        include("*.json")
+        include("jest.config.js")
+    })
+    inputs.files(fileTree("rewrite/src"))
+    inputs.files(fileTree("rewrite/test"))
+    outputs.files("rewrite/README.md") // A fake output entry; without it Gradle build caching doesn't work
+
+    dependsOn(tasks.named("npmInstall"))
```

Comment:
This is better described as `inputs.files(tasks.named("npmInstall"))` instead. 

Gradle best practices state that you should avoid explicit `dependsOn` statements and should instead prefer linking the outputs of the former into the inputs of the later.

The only exception to this is for lifecycle tasks, such as `check` below.